packages/mobile/src/import/saga.ts

```diff
@@ -154,6 +155,7 @@ export function* importBackupPhraseSaga({ phrase, useEmptyWallet }: ImportBackup
     yield call(storeMnemonic, mnemonic, account)
     // Set backup complete so user isn't prompted to do backup flow
     yield put(setBackupCompleted())
+    yield call(generateSignedMessage)
```

Comment:
i've updated this part since you commented, it was called more frequently before but now only in 2 places - when the account is initialised (create/import account flow) and the unlock account flow. ideally we'd only need to generate this message in the initialise account flow, the second case is for existing users who have just updated their app. i couldn't think of a central place to put this where both flows are covered, i think even from `assignAccountFromPrivateKey` it wouldn't cover the existing users, right?